pkg/app/piped/notifier/slack.go

```diff
@@ -350,6 +350,7 @@ func makeSlackDate(unix int64) string {
 	return fmt.Sprintf("<!date^%d^{date_num} {time_secs}|date>", unix)
 }
 
+// nolint:unparam
```

Comment:
Ignore "unparam" to resolve the following error.

> pkg/app/piped/notifier/slack.go:353:32: `truncateText` - `max` always receives `8` (unparam)
func truncateText(text string, max int) string {

It is impossible to remove the "always receives X" check currently.
cf. https://github.com/mvdan/unparam/issues/31